src/Lecture1.hs

```diff
@@ -50,7 +51,11 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+square :: Int -> Int
+square x = x * x
+
+sumOfSquares :: Int -> Int -> Int
+sumOfSquares x y = square x + square y
```

Comment:
Great solution! And an excellent idea of creating a separate helper function 🔝 

If you want to keep the `square` function local, you can use `where` instead:

```haskell
sumOfSquares :: Int -> Int -> Int
sumOfSquares x y = square x + square y
  where
    square :: Int -> Int
    square x = x * x
```

Or you can use the `^` operator to square numbers 🟦 

```haskell
sumOfSquares x y = x ^ 2 + y ^ 2
```